RetailCoder.VBE/Navigation/CodeExplorer/CodeExplorerItemViewModel.cs

```diff
@@ -69,17 +69,19 @@ public override int Compare(CodeExplorerItemViewModel x, CodeExplorerItemViewMod
                 if (SortOrder.TryGetValue(xNode.Declaration.DeclarationType, out xValue) &&
                     SortOrder.TryGetValue(yNode.Declaration.DeclarationType, out yValue))
                 {
-                    return xValue < yValue ? -1 : 1;
+                    if (xValue != yValue)
+                    { return xValue < yValue ? -1 : 1; }
                 }
-
-                return xNode.Declaration.DeclarationType < yNode.Declaration.DeclarationType ? -1 : 1;
             }
 
-            if (xNode.Declaration.Accessibility != yNode.Declaration.Accessibility)
+            var xNodeAcc = xNode.Declaration.Accessibility == Accessibility.Implicit ? Accessibility.Public : xNode.Declaration.Accessibility;
+            var yNodeAcc = yNode.Declaration.Accessibility == Accessibility.Implicit ? Accessibility.Public : yNode.Declaration.Accessibility;
+
+            if (xNodeAcc != yNodeAcc)
```

Comment:
you should explain why this is useful / necessary in a comment

Reply:
not sure what the names stand for either.

Reply:
it's not solely about the name. this is also about consistently applying the sort order to public members, whether they be implicitly public or explicitly

Reply:
Did ya'all figure this out or does it need a comment?

Reply:
both, I guess. Just because we figured it out in the end, that doesn't mean it doesn't need a comment :smile: